lib/Qi/Controller/Assets.pm

```diff
@@ -0,0 +1,44 @@
+package Qi::Controller::Assets;
+use Mojo::Base 'Mojolicious::Controller';
+use Qi::Model::Base;
+use SQL::Abstract;
+
+my $table = 'assets';
+
+sub get {
+  my $self = shift;
+  my $response = $self->req->json;
+  $self->stash('asset' => $self->base->select($response , $table));
+  $self->respond_to(
+    json => {json => $self->stash('asset')},
```

Comment:
No need to fetch from your model and then stash it, just use it directly.  Probably the most typical use of stash is to pass data to a template.  In this case, you're not using a template, you're just passing raw json.

So, L12-14 can be written as:

```
$self->render(json => $self->assets->select($response, $table);
```

There's actually a few additional ways to clean this up yet, too.  I'll get to that shortly.